test/app-tap/module_api.c

```diff
@@ -1075,7 +1075,7 @@ test_key_def_merge(struct lua_State *L)
 
 	/* Non-conventional prerequisite: list of known flags. */
 	uint32_t known_flags = key_part_def_known_flags();
-	assert(known_flags == BOX_KEY_PART_DEF_IS_NULLABLE);
+	assert(known_flags == (BOX_KEY_PART_DEF_IS_NULLABLE | BOX_KEY_PART_DEF_EXCLUDE_NULL));
```

Comment:
Please, update also the assert below about default values of flags. Otherwise direct initialization of the `flags` field with zero in the test cases below will not be a kind of 'legal'.